packages/core/src/__tests__/__e2e__/aira-announcer/AriaAnnouncer.cy.tsx

```diff
@@ -0,0 +1,31 @@
+import { AriaAnnouncerProvider } from "@jpmorganchase/uitk-core";
+
+describe("Given a AriaAnnouncerProvider", () => {
+  it("should not affect the document flow", () => {
+    cy.mount(
+      <div id="test-1" style={{ height: "100%", width: "100%" }}>
+        <AriaAnnouncerProvider>
+          <div style={{ height: "100%", width: "100%" }} />
+        </AriaAnnouncerProvider>
+      </div>
+    );
+
+    cy.document().then((doc) => {
+      const style = doc.createElement("style");
+      style.innerHTML = `
+                body, html {
+                    height: 100%;
+                    display: block;
+                    min-height: auto;
+                }
+                [data-cy-root] {
+                    height: 100%;
+                }
+            `;
```

Comment:
Do I need to clean this up, or are individual test files run in an isolated environment @joshwooding?